alongtheway.txt

```diff
@@ -9,7 +9,7 @@ Was anything surprising? Frustrating? Disappointing?
 
 WEEK ONE: NOAH
 
-Noah's journey is primarily one of obedience. His journey was mostly monotonous and uneventful, at least, until the flood. It was a long, long string of day after day walking in obedience in what God had called him. Noah's story is an example of "a long obedience in the same direction" (Friedrich Nietzsche) and I think this is the same type of obedience God is calling you and me to as well.
+Noah's journey is primarily one of obedience. His journey was mostly monotonous and uneventful, at least, until the flood. It was a long, long string of day after day walking in obedience in what God had called him. I think this is the same type of obedience God is calling you and me to as well.
```

Comment:
I see what you're saying. I like the phrase "a long obedience in the same direction" but your edit doesn't really change what I was saying.

BTW: thanks so much for doing this. You're doing an awesome job.

Sent from my iPhone

> On Jan 15, 2016, at 9:16 PM, Joshua Maxwell <notifications@github.com> wrote:
> 
> In alongtheway.txt:
> 
> > @@ -9,7 +9,7 @@ Was anything surprising? Frustrating? Disappointing?
> >  
> >  WEEK ONE: NOAH
> >  
> > -Noah's journey is primarily one of obedience. His journey was mostly monotonous and uneventful, at least, until the flood. It was a long, long string of day after day walking in obedience in what God had called him. Noah's story is an example of "a long obedience in the same direction" (Friedrich Nietzsche) and I think this is the same type of obedience God is calling you and me to as well.
> > +Noah's journey is primarily one of obedience. His journey was mostly monotonous and uneventful, at least, until the flood. It was a long, long string of day after day walking in obedience in what God had called him. I think this is the same type of obedience God is calling you and me to as well.
> Let's remove the Nietzsche quote since he's famous for inventing the phrase "God is dead." He is certainly a very complex person with struggles and pains like the rest of us, but I don't think it's a good idea to begin the entire bible study with a quote from him without devoting the rest of the study to unpacking his life and controversial stances.
> 
> —
> Reply to this email directly or view it on GitHub.
>